_posts/2016-02-04-chapter5-exercises.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+author: touchwick
+title: "Ian's Chapter 5 Excercises"
+---
+The following are the exercises for Chapter 5. The first:
+
+<iframe src="https://trinket.io/embed/python/80b4b9be9b" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+And the second:
+
+<iframe src="https://trinket.io/embed/python/5e427c3d20" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I don't think I did these correctly. I mean, I did do them correctly in the sense that I accomplished the task layed out in the instructions, but I didn't update the variable. I couldn't for the life of me get it to work. Instead, I made a list, appended each user input to the list, then had the program perform the required calculations when the user entered ```done```. Setting that up was very straightforward, which leaves me perplexed at my inability to get the variable update to behave. Anyway, live and learn, right? I assume this is something I will have to figure out eventually, and I'll get to the bottom of it. For the moment, though, the lists work great!
```

Comment:
See if you can replicate the error you were having in a duplicate trinket and drop the link here.  We'll dive in to figure out what was happening.

Good job using lists, but I want to make sure you understand how to accomplish this without them.